src/main/java/frc/robot/vision/VisionFilters.java

```diff
@@ -28,10 +29,13 @@ public static Filter<VisionData> isOnGround(double distanceFromGroundToleranceMe
 		return new Filter<>(visionData -> MathUtil.isNear(0, visionData.getEstimatedPose().getZ(), distanceFromGroundToleranceMeters));
 	}
 
-	public static Filter<AprilTagVisionData> isAprilTagHeightValid(double aprilTagRealHeightMeters, double aprilTagHeightToleranceMeters) {
+	public static Filter<AprilTagVisionData> isAprilTagHeightValid(double aprilTagHeightToleranceMeters) {
```

Comment:
```suggestion
	public static Filter<AprilTagVisionData> isAprilTagHeightValid(double aprilTagHeightToleranceMeters, Function<Integer, Double> getTagHeightInField) {
```

use that to get the height of the tag instead of using the Field stuff